common/src/main/java/com/microsoft/identity/common/internal/util/QueryParamsAdapter.java

```diff
@@ -81,16 +81,18 @@ public void write(final JsonWriter out, final List<Map.Entry<String, String>> qu
     }
 
     public static String _toJson(final List<Map.Entry<String, String>> extraQueryStringParameters) {
-        final Type listType = new TypeToken<List<Map.Entry<String, String>>>(){}.getType();
-        return mGson.toJson(extraQueryStringParameters, listType);
+        return mGson.toJson(extraQueryStringParameters, getListType());
     }
 
     public static List<Map.Entry<String, String>> _fromJson(final String jsonString) {
         if (TextUtils.isEmpty(jsonString)) {
             return new ArrayList<>();
         }
-        final Type listType = new TypeToken<List<Map.Entry<String, String>>>(){}.getType();
-        return mGson.fromJson(jsonString, listType);
+        return mGson.fromJson(jsonString, getListType());
+    }
+
+    public static Type getListType() {
```

Comment:
So this is a list that contains Maps of String, String...it's not just any list...we should javadoc this method and call it out and probably change the method name as well to reflect that.

Also does this method need to be public?